.eleventy.js

```diff
@@ -87,6 +89,11 @@ module.exports = function (eleventyConfig) {
       '*': 'en',
     },
   });
+  eleventyConfig.addPlugin(sitemap, {
+    sitemap: {
+      hostname: 'https://ceph.io',
```

Comment:
Got me thinking about the develop vs. prod versions of these sites. We may want some values like this to be dynamic, determined by the `NODE_ENV` the site was built for. So this would be environment-specific instead of hardcoded.

By extension we might want to consider adding `<meta name="robots" content="noindex nofollow">`, as one example, to the non-production environments 🤔 Worth another ticket?

Reply:
@adamduncan Something like this ?

```
eleventyConfig.addPlugin(sitemap, {
    sitemap: {
      hostname: process.env.NODE_ENV === 'production' ? 'https://ceph.io' : 'http://localhost:8080',
    },
  });
```

Reply:
Potentially in that ballpark, yep. Thinking more:

```
hostname: process.env.NODE_ENV === 'production' ? 'https://ceph.io' : 'http://develop.ceph.io',
```

...to cover the develop site. Not so concerned with running locally.

Though saying that, there will be some changes we need to make to the build scripts (and CI config) to ensure we're setting `NODE_ENV` for its respective envs. At the moment `build` just assumes `production`. 

I'd say merge this feature, and let's open a dedicated ticket for dev/prod env stuff?

Reply:
Yeah, ok